src/injections/js/bug1605611-maps.google.com-directions-time.js

```diff
@@ -45,38 +39,3 @@ document.addEventListener("DOMContentLoaded", () => {
   });
   mo.observe(document.body, moOptions);
 });
-
-// Step 2.
-const originalValueAsNumberGetter = Object.getOwnPropertyDescriptor(
```

Comment:
This is no longer required as the underlying issue was fixed in https://bugzilla.mozilla.org/show_bug.cgi?id=1283388